pkg/kubelet/network/exec/exec_test.go

```diff
@@ -51,7 +51,7 @@ func installPluginUnderTest(t *testing.T, vendorName string, plugName string) {
 	if err != nil {
 		t.Errorf("Failed to set exec perms on plugin")
 	}
-	writeStr := fmt.Sprintf("#!/bin/bash\necho -n $@ &> %s", path.Join(pluginDir, plugName+".out"))
+	writeStr := fmt.Sprintf("#!/bin/bash\nif [ \"$1\" == \"status\" ]; then echo -n '{ \"ipaddress\" : \"10.20.30.40\" }'; fi\n echo -n $@ &> %s", path.Join(pluginDir, plugName+".out"))
```

Comment:
can you turn this into a proper multi-line backtick-enclosed string?  it's complex enough now